examples/dxc/src/main.rs

```diff
@@ -0,0 +1,226 @@
+use bindings::Windows::Win32::{
+    Globalization::CP_UTF8,
+    Graphics::Hlsl::*,
+    System::{
+        Diagnostics::Debug::ERROR_FILE_NOT_FOUND,
+        SystemServices::{BOOL, PWSTR},
+    },
+};
+use libloading::{Library, Symbol};
+use std::path::Path;
+use std::rc::Rc;
+use windows::*;
+
+// Only exists in newer DXC headers
+const DXC_CP_UTF8: u32 = CP_UTF8;
```

Comment:
This will be cleaned/gone when https://github.com/microsoft/win32metadata/pull/474 lands, like: https://github.com/MarijnS95/windows-rs/commit/6e7d02d907abb88e1694b75840ec5bcff6bfed35